dpgen/data/surf.py

```diff
@@ -548,6 +549,13 @@ def run_vasp_relax(jdata, mdata):
     forward_files = ["POSCAR", "INCAR", "POTCAR"]
     backward_files = ["OUTCAR","CONTCAR"]
     forward_common_files = []
+    work_path_list = glob.glob(os.path.join(work_dir, "surf-*"))
+    task_format = {"fp" : "sys-*"}
+    for work_path in work_path_list :
+        symlink_user_forward_files(mdata=mdata, task_type="fp", work_path=work_path, task_format=task_format)
```

Comment:
Why not directly set format as `surf-*/sys-*`

Reply:
Because `surf-*` seems to align with the meaning of `work_path`.